File: algorithm/src/main/java/baekjoon/p10178/Main.java
package baekjoon.p10178;

import java.util.Locale;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int[] inputs = new int[]{
                    sc.nextInt(),
                    sc.nextInt()
            };

            System.out.println(solve(inputs));
        }
    }

    private static String solve(int[] inputs) {
        return String.format(
                Locale.getDefault(),
                "You get %d piece(s) and your dad gets %d piece(s).",
                inputs[0] / inputs[1],
                inputs[0] % inputs[1]
        );
    }

}

File: algorithm/src/main/java/baekjoon/p10178/Main.kt
package baekjoon.p10178

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val inputs = readLine()!!.split(" ").map { it2 -> it2.toInt() }
        val ret = "You get ${inputs[0] / inputs[1]} piece(s) and your dad gets ${inputs[0] % inputs[1]} piece(s)."
        println(ret)
    }
}